applications/crossbar/priv/couchdb/account/devices.json

```diff
@@ -13,6 +13,9 @@
         },
         "listing_by_macaddress": {
             "map": "function (doc) {if (doc.pvt_type != 'device' || doc.pvt_deleted) return; if(doc.mac_address) {emit(doc.mac_address, null)}}"
+        },
+        "listing_by_presence": {
```

Comment:
This name suggests, to me, the presence status of the device (ringing, answered, hungup). What other names would be more descriptive of what this view is showing?